src/ui/components/Collections/Collections.jsx

```diff
@@ -1,81 +1,215 @@
-import React from 'react';
+import React, { useState } from 'react';
 import axios from 'axios';
 import { useQuery } from 'react-query';
 import {
   Table,
   TableBody,
   TableCell,
   TableContainer,
-  TableHead,
-  TableRow
+  TableRow,
+  TablePagination
 } from '@material-ui/core';
 import ReactJson from 'react-json-view';
 import PropTypes from 'prop-types';
+import useStyles from './styles';
+import SortedTableHead from './SortedTableHead';
+import DeleteIcon from '@material-ui/icons/Delete';
+import CreateIcon from '@material-ui/icons/Create';
+import Button from '@material-ui/core/Button';
+import AddIcon from '@material-ui/icons/Add';
 
 const Collections = props => {
+  const classes = useStyles();
   const { selectedCollection } = props;
+  const [page, setPage] = useState(0);
+  const [rowsPerPage, setRowsPerPage] = useState(10);
+  const [order, setOrder] = useState('asc');
+  const [orderBy, setOrderBy] = useState('');
+
+  const handleChangePage = (event, newPage) => {
```

Comment:
The functions `handleChanhePage`, `handleRequestSort`, and `handleChangeRowsPerPage` should all use `useCallback` since they are passed into a react component.